examples/spec/integration/schedule_child_workflow_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'workflows/schedule_child_workflow'
+require 'workflows/hello_world_workflow'
+
+describe ScheduleChildWorkflow, :integration do
+  let(:cron_schedule) { "*/6 * * * *" }
+
+  it 'schedules a child workflow with a given cron schedule' do
+    child_workflow_id = 'schedule_child_test_wf-' + SecureRandom.uuid
+    workflow_id, run_id = run_workflow(
+      described_class,
+      child_workflow_id,
+      cron_schedule,
+      options: {
+        timeouts: { execution: 10 }
+      }
+    )
+
+    # Giving the parent workflow sufficient time to run and schedule the child
+    sleep 2
```

Comment:
Can we just wait for the completion here instead and only then check that the child workflow got scheduled? There should be some API that will allow to get details about a scheduled workflow before it is first executed. At least that's my guess…

Reply:
`DescribeWorkflowExecution`, which sits behind our `fetch_workflow_execution_info` and might do the trick…

Reply:
I couldn't find anything related to the cron schedule surfaced in DescribeWorkflowExecution, but I was able to find it in the event history in my revised approach. Let me know what you think.